src/index.html

```diff
@@ -6,8 +7,49 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+      src="images/imac.jpeg"
+      alt="imac"
+      class="card__img"
+      >
+      <h3
+      class="card__title">
+      APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
```

Comment:
Pay attention on code style. Fix all cases
```suggestion
      <img
        src="images/imac.jpeg"
        alt="imac"
        class="card__img"
      >
      <h3 class="card__title">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h3>
```